aggregator/src/aggregator/collection_job_driver.rs

```diff
@@ -60,7 +60,7 @@ pub struct CollectionJobDriver<B> {
 
 impl<R> CollectionJobDriver<R>
 where
-    R: Backoff + Clone + Send + Sync + 'static,
+    R: BackoffBuilder + Copy + Send + Sync + 'static,
```

Comment:
As above, I think the `Send + Sync` is redundant.

Reply:
Good catch, fixed in ee0aee9aef843d40c559cb6aca39006f4f819b4e